src/vm/internal/otable.h

```diff
@@ -1,15 +1,51 @@
 #ifndef OTABLE_H
 #define OTABLE_H
 
-/*
- * otable.c
- */
-void init_otable(void);
-void enter_object_hash(struct object_t *);
-void remove_object_hash(struct object_t *);
-void remove_precompiled_hashes(char *);
-struct object_t *lookup_object_hash(const char *);
-int show_otable_status(struct outbuffer_t *, int);
-struct array_t *find_ch_n(const char *s);
+#include<cassert>
+#include<memory>
```

Comment:
what is cassert and memory for?

Reply:
cassert is for an assertion that I have in the code. memory is for the std::shared_ptr.

Reply:
But there are no assert line in this file.   If it is not used here, you shouldn't include it here.

Reply:
There is a commented out assert in otable.cc.

Reply:
Why is it commented out then????